### ISSUE TITLE
Swagger UI broken for larger screen

### ISSUE THREAD

**Issue by vkmrishad** [tfranzel/drf-spectacular#1107]
**Describe the bug**
It was working last week, I reverted all changes and checked but still it looks broken.

![Speak-English-API](https://github.com/tfranzel/drf-spectacular/assets/18536460/fc17a6ed-f17b-4c21-a560-d9bbfb111e65)

When `.swagger-ui .opblock .opblock-summary-path` `max-width` is deselected, it is working fine.

<img width="407" alt="Screenshot 2023-11-13 at 7 46 31 PM" src="https://github.com/tfranzel/drf-spectacular/assets/18536460/6ca09e80-c482-46ed-89c6-d459c32eabfe">

<img width="614" alt="Screenshot 2023-11-13 at 7 43 08 PM" src="https://github.com/tfranzel/drf-spectacular/assets/18536460/d536e8ce-b318-4a85-b02e-53f7500ad45b">
<img width="1439" alt="Screenshot 2023-11-13 at 7 43 52 PM" src="https://github.com/tfranzel/drf-spectacular/assets/18536460/d55010db-da03-41ba-899a-fbc2a1cf0d71">

**Comment by tfranzel** [comment]
by default, spectacular only links the newest version of SwaggerUI:

https://github.com/tfranzel/drf-spectacular/blob/edce05317f7a8deec5a4c0940c70199e22be288a/drf_spectacular/settings.py#L78

We apply no UI changes. You might want wo pin to a different swagger version to test the bug. In any case, this is almost certainly an issue for the swagger UI project.

**Comment by vkmrishad** [comment]
@tfranzel Thank you, yes this is related to the swagger-ui-dist version.
I set it to an older version and working fine 👍
```
'SWAGGER_UI_DIST': 'https://cdn.jsdelivr.net/npm/swagger-ui-dist@5.9.3',
```

**Comment by tfranzel** [comment]
closed in favor of #1108
